Source/SlideOutable.swift

```diff
@@ -437,16 +437,18 @@ open class SlideOutable: ClearContainerView {
         activeScroll.frame.size.height += antiBounce
         activeScroll.contentInset.bottom += antiBounce
         
-        // Animate to new height
-        UIView.animate(withDuration: 0.5, delay: 0,
-                       usingSpringWithDamping: 0.8,
-                       initialSpringVelocity: velocity.flatMap { abs($0 / (currentOffset - offset)) } ?? 1,
```

Comment:
Previously we had calculated the initial velocity to match users gesture with animation. Does `UIViewPropertyAnimator` do this out of the box? 😮

Reply:
Ping!